src/client/Microsoft.Identity.Client/ConfidentialClientApplication.cs

```diff
@@ -183,6 +184,11 @@ public sealed partial class ConfidentialClientApplication
         /// </summary>
         public X509Certificate2 Certificate { get; }
 
+        /// <summary>
+        /// Send X5C with each request
+        /// </summary>
+        public bool SendX5C { get; }

```

Comment:
Does this need to be here? It is set but unused anywhere. Plus this class is public.

Reply:
Good catch. Removed